src/components/ui.js

```diff
@@ -124,3 +128,196 @@ export const BarNavigationWrapper = posed(styled(Box)`
     },
   },
 });
+
+export const HeaderNavigationItemWrapper = styled('span')`
+  position: relative;
+
+  &::before {
+    left: 50%;
+    top: 17px;
+    content: '^';
+    font-weight: 700;
+    position: absolute;
+    opacity: ${({ active }) => ifElse(active, 1, 0)};
+    color: ${themeGet('colors.mainOrange', 'white')}; 
+  }
+`;
+
+export const BarNavigationItemWrapper = posed(styled('div')`
+  position: relative;
+
+  &::before {
+    top: 0px;
+    left: -15px;
+    content: '>';
+    font-weight: 700;
+    position: absolute;
+    opacity: ${({ active }) => ifElse(active, 1, 0)};
+    color: ${themeGet('colors.mainOrange', 'white')}; 
+  }
+`)({
+  opened: { opacity: 1 },
+  closed: { opacity: 0 },
+});
+
+// Contacts form styles
+export const Title = styled(Flex)`
+  height: 55px; 
+  color: #1e1b18; 
+  font-size: 24px; 
+  font-weight: bold; 
+  position: relative; 
+  font-family: Roboto;  /*TODO: connect this font*/
+  align-items: center; 
+  flex-direction: column; 
+  justify-content: center;
+`;
+
+export const Line = styled.div`
+  bottom: 0; 
+  width: 47.5px; 
+  height: 4.5px; 
+  position: absolute; 
+  border-radius: 10px; 
+  background-color: #f18701;
+`;
+
+export const FormContainer = styled.div`
+  display: flex;
+  margin-top: 22px;
+  align-items: center;
+  flex-direction: column;  
+`;
+
+export const FormInput = styled(Input)`
+  border: 0;
+  width: 262px;
+  font-size: 12px;
+  border-radius: 0;
+  margin-bottom: 15px;
+  font-family: "Roboto";
+  border-bottom: 1.5px solid #f18701; 
+  border-color: ${({ invalid }) => invalid && 'red'};
+`;
+
+export const FormFieldContainer = styled.div`
+  position: relative;
+
+  &::after {
+    left: 0;
+    top: 22px;
+    color: red;
+    font-size: 10px;
+    position: absolute;
+    white-space: nowrap;
+    content: '${({ text }) => text}';
+  }
+`;
+
+export const FormTextArea = styled(TextArea)`
+  width: 262px;
+  resize: none;
+  height: 133.5px;
+  font-size: 12px;
+  font-family: "Roboto";
+  border-bottom: 1.5px solid #f18701;
+`;
+
+export const FormButtonsContainer = styled(Flex)`
```

Comment:
instead this component use <Flex mt={33} mb={33} justifyContent='space-between'>content here</Flex>